src/details/ArborX_DetailsBruteForceImpl.hpp

```diff
@@ -24,118 +24,115 @@ namespace Details
 {
 struct BruteForceImpl
 {
-  template <class ExecutionSpace, class Primitives, class BoundingVolumes,
-            class Bounds>
+  template <class ExecutionSpace, class Values, class IndexableGetter,
+            class Nodes, class Bounds>
```

Comment:
I am not a big fan of that name "Nodes" but I understand we need to distinguish form the user-provided input and I don't have a better idea at that time.